src/content-helper/editor-sidebar/smart-linking/component.tsx

```diff
@@ -351,6 +352,28 @@ export const SmartLinkingPanel = ( {
 			return true;
 		} );
 
+		// Exclude Smart Links text and URLs that are already in the post, to avoid duplicates.
+		links = links.filter( ( link ) => {
+			return ! smartLinks.some( ( sl ) => {
+				if ( sl.href === link.href ) {
+					// eslint-disable-next-line no-console
+					console.warn( `PCH Smart Linking: Skipping duplicate link: ${ link.href }` );
```

Comment:
Do you think we should leave these `console.warn` calls?

Reply:
That's something I have been thinking about myself. They are useful to understand why the API call returns links that are not shown in the review dialog, however I believe that for 99.9% of our users this information will have no interest. But on the other hand, the browser console is also hidden most of the time.
The way I see it being useful is if a user has a report that the Smart Linking isn't returning any suggestions - that way we could ask them (or verify ourselves) to check the console for skipped suggestions. 

What do you think?

Reply:
I can see the usefulness of this. Until we've got some debug tools, let's leave it.